src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -799,7 +831,10 @@ ActStatus ActionHandlers::cs_req_to_sgw(SM::ControlBlock& cb)
     memset(&cs_msg, 0, sizeof(cs_msg));
 	cs_msg.msg_type = create_session_request;
 	cs_msg.ue_idx = ue_ctxt->getContextID();
-	
+	if(mme_cfg->dns_config.dns_flag == 1)
+   	{
+		cs_msg.sgw_ip = ue_ctxt->getSgwCtrlPIP();
+	}
```

Comment:
We have same block at at line 893 to 896

Reply:
Sorry, kindly explain what need to done

Reply:
removed the duplicate block